decidim-core/app/packs/stylesheets/decidim/_login.scss

```diff
@@ -1,6 +1,11 @@
+$facebook_background: #1877f2;
```

Comment:
This value come from FB guidelines? We should tune this a bit because it's not accessible: 

![image](https://github.com/user-attachments/assets/64fb3aef-9337-4979-a267-7184cdb7c172)

Reply:
Yes: https://www.figma.com/design/1oiYPbgbpYCXDlizz9bSxs/Internal-Decidim?node-id=105-11460&t=bdviaa4fmP9CiZVL-0. Whats the guidance here on tuning it?

Reply:
We've discussed about this and we agree with using what FB has on their login page: #0866ff 
According to https://webaim.org/resources/contrastchecker/ it passes the contrast checker for WCAG AA